src/screens/UserPortal/Chat/Chat.tsx

```
@@ -57,6 +60,12 @@ interface InterfaceContactCardProps {
 export default function chat(): JSX.Element {
   const { t } = useTranslation('translation', { keyPrefix: 'userChat' });
   const { t: tCommon } = useTranslation('common');
+  const params = useParams<{ orgId: string }>();
+  const organizationId = params.orgId!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)